apps/cli/src/node/docker-compose-anvil.yaml

```diff
@@ -32,9 +32,10 @@ services:
                 "http://anvil:8545",
                 "--private-key",
                 "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80",
-                "0x9E32e06Fd23675b2DF8eA8e6b0A25c3DF6a60AbC",
+                "0x4C11C7F82D6D56a726f9B53dd99af031AFd86BB6",
                 "deployContracts(address,address,bytes32,bytes32)",
                 "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+                "720",
```

Comment:
I know this should not be hardcoded here. Not caring much for now, because the application deployment process will change to use the rollups-node v2